src/request/arrive_before.rs

```diff
@@ -259,6 +259,25 @@ where
         Some(new_criteria)
     }
 
+    fn stay_in(&self, trip: &Data::Trip, criteria: &Criteria) -> Option<(Data::Trip, Criteria)> {
+        let previous_trip = self
+            .transit_data
+            .stay_in_previous(trip, self.real_time_level)?;
+        let mission = self.transit_data.mission_of(&previous_trip);
+        let last_position = self.transit_data.last_on_mission(&mission);
+        let (arrival_time_at_last_stop, load) = self
+            .transit_data
+            .arrival_time_of(&previous_trip, &last_position);
+        let new_criteria = Criteria {
+            time: arrival_time_at_last_stop,
+            nb_of_legs: criteria.nb_of_legs + 1,
+            fallback_duration: criteria.fallback_duration,
+            transfers_duration: criteria.transfers_duration,
+            loads_count: criteria.loads_count.add(load),
```

Comment:
the load inside the stay-in is not present in the loads data. So I think we should just :
```suggestion
            loads_count: criteria.loads_count,
```